superset-frontend/src/views/CRUD/annotation/AnnotationList.tsx

```diff
@@ -259,9 +259,11 @@ function AnnotationList({
             <span>{t('Annotation Layer %s', annotationLayerName)}</span>
             <span>
               {hasHistory ? (
-                <Link to="/annotationlayer/list/">Back to all</Link>
+                <Link to="/annotationlayermodelview/list/">
+                  {t('Back to all')}
+                </Link>
               ) : (
-                <a href="/annotationlayer/list/">Back to all</a>
+                <a href="/annotationlayermodelview/list/">{t('Back to all')}</a>
```

Comment:
Can you explain the URI changes here? Not sure if this is bycatch (explanation would be helpful), or somehow related to the translation effort.

Reply:
I don't know how it happened :( Changed back.